packages/cli/src/commands/worker.ts

```diff
@@ -43,6 +46,14 @@ export class Worker extends BaseCommand {
 
 	static jobQueue: JobQueue;
 
+	readonly workerId: string;
+
+	constructor(argv: string[], cmdConfig: IConfig) {
+		super(argv, cmdConfig);
+		this.workerId = generateHostInstanceId('worker');
```

Comment:
you could move this line to `init`, and then use `this.logger` instead of `LoggerProxy`

Reply:
if it's in init then worker id can't be a readonly property any more